docker-compose.yaml

```diff
@@ -8,6 +8,8 @@ services:
       - "SAAS_BASE_URL=https://meshery.layer5.io"
       - "ADAPTER_URLS=meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-octarine:10003 meshery-nsm:10004"
       - "EVENT=mesheryLocal"
+    volumes:
+      - $HOME/.meshery/config:/home/appuser/.meshery/config
```

Comment:
Would it be helpful for other potential use cases to mount $HOME/.meshery itself?

Reply:
if we do that, we will also loose the yaml file